app/src/main/java/com/onboarding/calculator/util/ConstantsUtils.java

```diff
@@ -20,4 +20,15 @@
     public static final int CORRECTION_FACTOR = 1;
     public static final double DEFAULT_RESULT = 0.0;
 
+    //TEST
```

Comment:
remove this comment line, because you have the sufix "TEST" in the constants

Reply:
Solved!